File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md

```diff
@@ -399,250 +268,161 @@ The following code example demonstrates how to edit an existing connector in a P
+    //Set the line format for the connector
+    connector.LineFormat.DashStyle = LineDashStyle.DashDotDot;
+    //Disconnect the end connection of the connector if end point get connected
+    if (connector.EndConnectedShape != null)
+        connector.EndDisconnect();
+    //Insert a triangle shape into slide
+    IShape triangle = slide.Shapes.AddShape(AutoShapeType.IsoscelesTriangle, 600, 500, 150, 150);
+    //Declare the end connection site index
+    int connectionSiteIndex = 4;
+    //Reconnect the end point of connector with triangle shape if its connection site count is greater than 4
+    if (connectionSiteIndex < triangle.ConnectionSiteCount)
+        connector.EndConnect(triangle, connectionSiteIndex);
+    //Save the PowerPoint Presentation as stream
+    FileStream outputStream = new FileStream("Connector.pptx", FileMode.Create);
+    pptxDoc.Save(outputStream);
 }
-
 {% endhighlight %}
 
 {% highlight c# tabtitle="Xamarin" %}
-
 //"App" is the class of Portable project.
 Assembly assembly = typeof(App).GetTypeInfo().Assembly;
 Stream inputStream = assembly.GetManifestResourceStream("SampleBrowser.Presentation.Samples.Template.Sample.pptx");
-
 //Opens the loaded PowerPoint file
 using (IPresentation pptxDoc = Presentation.Open(inputStream))
-
 {
-
-   //Get the first slide of a PowerPoint file
-   ISlide slide = pptxDoc.Slides[0];
-
-   //Get the connector from a slide
-   IConnector connector = slide.Shapes[2] as IConnector;
-
-   //Set the begin cap for the connector
-   connector.LineFormat.BeginArrowheadStyle = ArrowheadStyle.ArrowOpen;
-
-   //Set the line format for the connector
-   connector.LineFormat.DashStyle = LineDashStyle.DashDotDot;
-
-   //Disconnect the end connection of the connector if end point get connected
-   if (connector.EndConnectedShape != null)
-      connector.EndDisconnect();
-
-   //Insert a triangle shape into slide
-   IShape triangle = slide.Shapes.AddShape(AutoShapeType.IsoscelesTriangle, 600, 500, 150, 150);
-
-   //Declare the end connection site index
-   int connectionSiteIndex = 4;
-
+    //Get the first slide of a PowerPoint file
+    ISlide slide = pptxDoc.Slides[0];
+    //Get the connector from a slide
+    IConnector connector = slide.Shapes[2] as IConnector;
+    //Set the begin cap for the connector
+    connector.LineFormat.BeginArrowheadStyle = ArrowheadStyle.ArrowOpen;
+    //Set the line format for the connector
+    connector.LineFormat.DashStyle = LineDashStyle.DashDotDot;
+    //Disconnect the end connection of the connector if end point get connected
+    if (connector.EndConnectedShape != null)
+        connector.EndDisconnect();
+    //Insert a triangle shape into slide
+    IShape triangle = slide.Shapes.AddShape(AutoShapeType.IsoscelesTriangle, 600, 500, 150, 150);
+    //Declare the end connection site index
+    int connectionSiteIndex = 4;
    //Reconnect the end point of connector with triangle shape if its connection site count is greater than 4
```

Comment:
Alignment is not proper in this code snippet.

Reply:
@Suriya-Balamurugan -Done